www/src/Pages/SettingsPage.jsx

```diff
@@ -244,22 +244,23 @@ const HOTKEY_ACTIONS = [
 	{ labelKey: 'hotkey-actions.load-profile-2', value: 16 },
 	{ labelKey: 'hotkey-actions.load-profile-3', value: 17 },
 	{ labelKey: 'hotkey-actions.load-profile-4', value: 18 },
-	{ labelKey: 'hotkey-actions.l3-button', value: 19 },
-	{ labelKey: 'hotkey-actions.r3-button', value: 20 },
-	{ labelKey: 'hotkey-actions.touchpad-button', value: 21 },
-	{ labelKey: 'hotkey-actions.reboot-default', value: 22 },
-	{ labelKey: 'hotkey-actions.b1-button', value: 23 },
-	{ labelKey: 'hotkey-actions.b2-button', value: 24 },
-	{ labelKey: 'hotkey-actions.b3-button', value: 25 },
-	{ labelKey: 'hotkey-actions.b4-button', value: 26 },
-	{ labelKey: 'hotkey-actions.l1-button', value: 27 },
-	{ labelKey: 'hotkey-actions.r1-button', value: 28 },
-	{ labelKey: 'hotkey-actions.l2-button', value: 29 },
-	{ labelKey: 'hotkey-actions.r2-button', value: 30 },
-	{ labelKey: 'hotkey-actions.s1-button', value: 31 },
-	{ labelKey: 'hotkey-actions.s2-button', value: 32 },
-	{ labelKey: 'hotkey-actions.a1-button', value: 33 },
-	{ labelKey: 'hotkey-actions.a2-button', value: 34 },
+	{ labelKey: 'hotkey-actions.next-profile', value: 19 },
+	{ labelKey: 'hotkey-actions.l3-button', value: 20 },
+	{ labelKey: 'hotkey-actions.r3-button', value: 21 },
+	{ labelKey: 'hotkey-actions.touchpad-button', value: 22 },
+	{ labelKey: 'hotkey-actions.reboot-default', value: 23 },
+	{ labelKey: 'hotkey-actions.b1-button', value: 24 },
+	{ labelKey: 'hotkey-actions.b2-button', value: 25 },
+	{ labelKey: 'hotkey-actions.b3-button', value: 26 },
+	{ labelKey: 'hotkey-actions.b4-button', value: 27 },
+	{ labelKey: 'hotkey-actions.l1-button', value: 28 },
+	{ labelKey: 'hotkey-actions.r1-button', value: 29 },
+	{ labelKey: 'hotkey-actions.l2-button', value: 30 },
+	{ labelKey: 'hotkey-actions.r2-button', value: 31 },
+	{ labelKey: 'hotkey-actions.s1-button', value: 32 },
+	{ labelKey: 'hotkey-actions.s2-button', value: 33 },
+	{ labelKey: 'hotkey-actions.a1-button', value: 34 },
+	{ labelKey: 'hotkey-actions.a2-button', value: 35 },
```

Comment:
As earlier, you shouldn't change the existing enum values, so most of this diff should go away.

Reply:
Understood. I will change the enum values back and fix the spacing. Thank you for the feedback.